frontend-react/src/pages/eventPages/EventSettingListPage.tsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from 'react'
+import { useApiClient } from "@/hooks/useApiClient"
+import { SelectOption } from "@/types"
+
+export default function EventList() {
+  const [recruitments, setRecruitments] = useState<SelectOption[]>([])
+  const [errors, setErrors] = useState<string[]>([])
+  const apiClient = useApiClient()
+
+  const getRecruitment = async () => {
+    setErrors([])
+    try {
+      const res = await apiClient.get('/recruitments')
+      setRecruitments(res.data.data)
+    } catch {
+      setErrors(["イベントを表示できませんでした。"])
+    }
+  }
+
+  useEffect(() => {
+    getRecruitment()
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
```

Comment:
超細かいのですが以下のような順番で書いた方が関数定義の部分とuseEffectの部分が入れ子にならなくてスッキリして見える
```suggestion
  useEffect(() => {
    getRecruitment()
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, [])
  
  const getRecruitment = async () => {
    setErrors([])
    try {
      const res = await apiClient.get('/recruitments')
      setRecruitments(res.data.data)
    } catch {
      setErrors(["イベントを表示できませんでした。"])
    }
  }
```

Reply:
あっ、そうなんですね！
今まで関数呼び出しは、関数定義の後に書いていました
次から「関数呼び出し→関数定義」の順番にします